src/test/dynamo/Beyonce.test.ts

```diff
@@ -13,7 +13,7 @@ import {
   SongModel,
   table,
 } from "./models"
-import { setup } from "./util"
```

Comment:
Up till now we've had all our unit tests in a single file. That file was getting big. And so, this commit splits out query related tests into their own file (since we'll be adding scan related tests later).